samples/cext-trey-research-csharp/TeamsApp/teamsapp.local.yml

```diff
@@ -0,0 +1,50 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Microsoft-Teams-Samples:cext-trey-research-csharp-csharp
+
+provision:
+  # Set TAB_DOMAIN and TAB_ENDPOINT for local launch
+  - uses: script 
+    with:
+      run:
+        echo "::set-teamsfx-env TAB_DOMAIN=localhost:44302";
+        echo "::set-teamsfx-env TAB_ENDPOINT=https://localhost:44302";
```

Comment:
These environment variables are not used, so we should remove them.

Reply:
Fixed